site/content/integrate/admin-guide/admin-webhooks-outgoing/_index.md

```diff
@@ -0,0 +1,151 @@
+
+5. If you specified one or more trigger words on the previous step, choose when to trigger the outgoing webhook.
+
+ - If the first word of a message matches one of the trigger words exactly, or
+ - If the first word of a message starts with one of the trigger words.
+
+6. Finally, set one or more callback URLs that HTTP POST requests will be sent to, then select **Save**. If the URL is private, add it as a [trusted internal connection](https://about.mattermost.com/default-allow-internal-connections-settings-documentation).
+7. On the next page, copy the **Token** value. This will be used in a later step.
+
+![image](images/outgoing_webhooks_token.png)
+
+8. Next, write your external application. Include a function, which receives HTTP POST requests from Mattermost. The function should look something like this:
+
+```
+      Content-Length: 244
+      User-Agent: Go 1.1 package http
+      Host: localhost:5000
+      Accept: application/json
+      Content-Type: application/x-www-form-urlencoded
+
+      channel_id=hawos4dqtby53pd64o4a4cmeoo&
+      channel_name=town-square&
+      team_domain=someteam&
+      team_id=kwoknj9nwpypzgzy78wkw516qe&
+      post_id=axdygg1957njfe5pu38saikdho&
+      text=some+text+here&
+      timestamp=1445532266&
+      token=zmigewsanbbsdf59xnmduzypjc&
+      trigger_word=some&
+      user_id=rnina9994bde8mua79zqcg5hmo&
+      user_name=somename
+```
+
+If your integration sends back a JSON response, make sure it returns the `application/json` content-type.
+
+9. Add a configurable *MATTERMOST_TOKEN* variable to your application and set it to the **Token** value from step 7. This value will be used by your application to confirm the HTTP POST request came from Mattermost.
+10. To have your application post a message back to `town-square`, it can respond to the HTTP POST request with a JSON response such as:
+
+```
+  {"text": "
+  | Component  | Tests Run   | Tests Failed                                   |
+  |:-----------|:------------|:-----------------------------------------------|
+  | Server     | 948         | :white_check_mark: 0                           |
+  | Web Client | 123         | :warning: [2 (see details)](http://linktologs) |
+  | iOS Client | 78          | :warning: [3 (see details)](http://linktologs) |
+  "}
+```
+
+which would render in Mattermost as:
+
+![image](images/webhooksTable.png)
+
+11. You're all set! See [developer documentation](https://developers.mattermost.com/integrate/outgoing-webhooks) for details on what parameters are supported by outgoing webhooks. For instance, you can override the username and profile picture the messages post as, or specify a custom post type when sending a webhook message for use by [plugins](https://about.mattermost.com/default-plugins).
```

Comment:
https://about.mattermost.com/default-plugins should be replace by an internal link